.github/scripts/modules_test.sh

```diff
@@ -97,6 +97,9 @@ for f in spyder/*/*/*/*.py; do
     if [[ $f == spyder/plugins/ipythonconsole/widgets/__init__.py ]]; then
         continue
     fi
+    if [[ $f == spyder/plugins/profiler/widgets/profilergui.py ]]; then
```

Comment:
Why is it not possible to run this test anymore?

Reply:
I don't know. It was segfaulting. When I asked you some time ago you told me to comment it.